examples/archive.html

```diff
@@ -17,6 +17,19 @@
     <script src="../dist/leaflet.distortableimage.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
     <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="./js/archive.js"></script>
+  </head>
+
+  <body style="margin:0;">
+    
+    <div class="modal fade" id="welcomeModal" aria-labelledby="exampleModalLabel" tabindex="-1" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
+        <div class="modal-content bg-light">
+          <div class="modal-header d-block">
+            <div class="d-flex justify-content-between align-items-center">
+              <h2 class="modal-title">Welcome to MapKnitter Lite</h2>
+              <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close" ></button>
+
```

Comment:
Hey @Aarav238, great work so far, this snippet seems to be in the wrong place, could you go over it again? Thanks!

Reply:
I haven't understood what is the issue here @7malikk . Please elaborate little bit more.

Reply:
Hey there @Aarav238 If you take a look at this snippet of your file

https://github.com/publiclab/Leaflet.DistortableImage/blob/b9511d3533fac214a97fe5caee1f3d58bc5e8933/examples/archive.html#L21-L44

You'd notice that your file has two `</head>` tags at lines 21 and 35 respectively 
You'd also notice there are two `<body>` tags at lines 23 and 36 respectively

This further implies that there are duplicate codes within your file.

Would you be willing to make a correction?

Reply:
Yeah sure.